openshift/system/.env

```diff
@@ -1,6 +1,8 @@
 # S3 access
 AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
+AWS_ROLE_ARN=
+AWS_WEB_IDENTITY_TOKEN_FILE=
```

Comment:
`AWS_REGION` would be defined here as well.

Reply:
And we were missing some others too. Fixed on c2d9b5d. Thank you!